test/test_helper.exs

```diff
@@ -6,5 +6,9 @@ defmodule TestHelper do
     def read(filename) do
       File.read!("test/fixtures/#{filename}")
     end
+
+    def dummy do
+      File.read!("test/fixtures/dummy.json")
```

Comment:
Gotcha, yah the annoying ones were the associations anyway, especially with something like Lists where there are a ton.